### DISCUSSION TITLE
How to .engine syntax of express in fastify

### DISCUSSION THREAD

**Discussion by vijayShankarfastify** [fastify/fastify#3433] [General] [answered]
var expres= require('expres');
var app = expres();
var mustacheExpress = require('mustache-express');
app.engine('html', mustacheExpress());

How to convert above express code in to fastify?

**Comment by zekth** [comment, marked as answer]
This is what you're looking for : https://github.com/fastify/point-of-view

**Reply by vijayShankarfastify** [reply]
Ok thank you.

 express.set('view engine', 'html');
 const router = express.Router({});

How to achieve above express set and Router functions in fastify.

**Reply by zekth** [reply]
Please read the documentations of Fastify and point-of-view. Those will explain more than anything else.

**Reply by vijayShankarfastify** [reply]
any existing git hub project where we used those api methods using fastify.
